ddtrace/appsec/processor.py

```diff
@@ -28,9 +30,20 @@
 log = get_logger(__name__)
 
 
-def _no_cookies(data):
-    # type: (Dict[str, str]) -> Dict[str, str]
-    return {key: value for key, value in data.items() if key.lower() not in ("cookie", "set-cookie")}
+def _transform_headers(data):
+    # type: (Dict[str, str]) -> Dict[str, Union[str, List[str]]]
```

Comment:
@brettlangdon the Type checker does not really like this situation, is there a way to make it work? Rignt now, it fails with
```
ddtrace/appsec/processor.py:42: error: List item 0 has incompatible type "Union[str, List[str]]"; expected "str"  [list-item]
ddtrace/appsec/processor.py:43: error: Item "str" of "Union[str, List[str]]" has no attribute "append"  [union-attr]
```